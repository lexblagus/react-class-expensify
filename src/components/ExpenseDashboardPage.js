import React from 'react';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from './ExpensesSummary';
import ExpenseList from './ExpenseList';


const ExpenseDashboardPage = () => (
	<div>
		<section className="section">
			<div className="container">
				<h2 className="title is-2">View</h2>
				<div className="card">
					<div className="card-content">
						<ExpenseListFilters />
						<ExpensesSummary />
						<ExpenseList />
					</div>
				</div>
			</div>
		</section>
	</div>
);

export default ExpenseDashboardPage;
