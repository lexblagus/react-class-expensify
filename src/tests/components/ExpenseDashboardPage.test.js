import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage  from '../../components/ExpenseDashboardPage';
import storeDummyData from '../fixtures/storeDummyData';


test('Should render ExpenseDashboardPage', ()=>{
	//console.log( {...storeDummyData[0]} );
	
	const wrapper = shallow(
		<ExpenseDashboardPage />
	);
	
	expect(wrapper).toMatchSnapshot();
});

