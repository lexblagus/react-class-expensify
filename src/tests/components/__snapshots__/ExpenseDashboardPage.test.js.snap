// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseDashboardPage 1`] = `
<div>
  <section
    className="section"
  >
    <div
      className="container"
    >
      <h2
        className="title is-2"
      >
        View
      </h2>
      <div
        className="card"
      >
        <div
          className="card-content"
        >
          <Connect(ExpenseListFilters) />
          <Connect(ExpenseList) />
        </div>
      </div>
    </div>
  </section>
</div>
`;
